docs/advanced-topics/metrics.rst

```diff
@@ -12,38 +16,62 @@ Users are able to execute workflows on various platforms using Dockstore's Launc
+Record workflow execution metrics on your platform
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to collect metrics for executions of public Dockstore workflows that occur on your platform. The mandatory metrics that Dockstore requires are the date of execution and the execution status. 
+If they are available, you may also collect additional metrics such as how long it took the workflow to run, CPU requirements, memory requirements, etc.
+
+Format the workflow metrics for Dockstore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The metrics that you collect may not be in the format that the Dockstore API expects. You have to format them so they follow Dockstore's schema, which you can view in the :ref:`Submit Workflow Metrics to Dockstore<Submit executions>` section.
 
-Platform owners are able to submit metrics for workflow executions, retrieve submitted executions, and update submitted executions.
+For example, execution dates are expected to be in ISO 8601 UTC date format and there are a defined set of execution statuses that can be submitted.
 
-.. note:: Submitting metrics for workflow executions, and retrieving and updating submitted executions are only available for admins, curators, and platform partners. If you're a platform owner and you don't have a platform partner user, please contact us via our `GitHub <https://github.com/dockstore/dockstore/issues>`_ issues or open a helpdesk ticket on `Discourse <https://discuss.dockstore.org/>`_ and we will help you get set up.
+.. note::
+   If you are unable to format the workflow metrics to match Dockstore's schema or you would like to try submitting metrics without continuing with the following steps, 
+   contact us via our `GitHub <https://github.com/dockstore/dockstore/issues>`_ issues or open a helpdesk ticket on `Discourse <https://discuss.dockstore.org/>`_ and we can help get your metrics into Dockstore.
+   
+   This may involve you providing us with a file of metrics, such as a CSV file, and Dockstore will process the file and ingest the metrics into our system.
 
+Register a platform partner user on Dockstore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-How do I submit workflow metrics?
----------------------------------
+Only Dockstore admins, curators, and platform partners can :ref:`submit metrics<Submit executions>` for workflow executions, and :ref:`retrieve and update submitted executions<Viewing And Updating Submitted Workflow Metrics>`.
+If you're a platform owner and you don't have a platform partner user on Dockstore, please:
+
+1. :doc:`Create a Dockstore account </getting-started/register-on-dockstore>`, if you haven't already. You will use the credentials of this user to submit Dockstore metrics.
+2. Contact us via our `GitHub <https://github.com/dockstore/dockstore/issues>`_ issues or open a helpdesk ticket on `Discourse <https://discuss.dockstore.org/>`_ and we will give your Dockstore user a platform partner role.
```

Comment:
I'd suggest only a helpdesk ticket, and not mention a GitHub issue -- the paranoid side of me doesn't want to potentially identify public elevated privilege users.